pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -288,6 +288,12 @@ func (m *kubeGenericRuntimeManager) makeMounts(opts *kubecontainer.RunContainerO
 			Propagation:    v.Propagation,
 		}
 
+		// Docker on Windows does some path fixups that were not carried into ContainerD.
+		// If using ContainerD, make sure paths inside the container have 'c:' and slashes facing the right way.
+		// This check would need to be changed later to support a Linux container on a Windows node
```

Comment:
Can you share more about this final line of the comment? Want to be sure I'm fully understanding.